src/applications/mhv-medications/components/PrescriptionDetails/VaPrescription.jsx

```diff
@@ -138,28 +138,20 @@ const VaPrescription = prescription => {
               Quantity
             </h3>
             <p>{validateField(prescription.quantity)}</p>
-          </div>
-          {isDisplayingDocumentation &&
-            prescription.cmopNdcNumber && (
-              <div className="vads-u-border-top--1px vads-u-border-color--gray-lighter vads-u-margin-bottom--3">
-                <h2
-                  className="vads-u-margin-top--3"
-                  data-testid="refill-History"
-                >
-                  Documentation
-                </h2>
+            {isDisplayingDocumentation &&
+              refillHistory[0].cmopNdcNumber && (
```

Comment:
cmop doesnt always appear on the first refilHistory item, this will have to cover a history of multiple items. Unless this is intentional and we only want to show the first refill history documentation.

Reply:
refrence:
user: elena(id: 17621060)
rx id: 21997544

Reply:
this is intended and is working as expected. I've reached out to the team about the other fields and we will address it in another ticket if need be. Thanks for pointing this out.